scripts/apps/publish/views/email-config.html

```diff
@@ -1,6 +1,10 @@
 <div class="field">
     <label for="recipients" translate>Recipients</label>
-    <input type="text" id="recipients" placeholder="{{:: 'Recipients'|translate }}"
-      ng-model="destination.config.recipients" required
+    <input type="text" id="recipients" placeholder="{{:: 'Recipients (visible to everybody)'|translate }}"
```

Comment:
@mdhaman added a check to see if at least one of the field is filled, thank for the feedback